docs/getting_started.rst

```diff
@@ -0,0 +1,139 @@
+the `GitHub issue tracker <https://github.com/zarr-developers/zarr-python/issues>`_. See
+:doc:`contributing` for further information about contributing to Zarr.
+
+Projects using Zarr
+-------------------
+
+If you are using Zarr, we would `love to hear about it
+<https://github.com/zarr-developers/community/issues/19>`_.
+
+Acknowledgments
+---------------
+
+The following people have contributed to the development of Zarr by contributing code,
+documentation, code reviews, comments and/or ideas:
+
+:user:`Alistair Miles <alimanfoo>`
+:user:`Altay Sansal <tasansal>`
+:user:`Anderson Banihirwe <andersy005>`
+:user:`Andrew Fulton <andrewfulton9>`
+:user:`Andrew Thomas <amcnicho>`
+:user:`Anthony Scopatz <scopatz>`
+:user:`Attila Bergou <abergou>`
+:user:`BGCMHou <BGCMHou>`
+:user:`Ben Jeffery <benjeffery>`
+:user:`Ben Williams <benjaminhwilliams>`
+:user:`Boaz Mohar <boazmohar>`
+:user:`Charles Noyes <CSNoyes>`
+:user:`Chris Barnes <clbarnes>`
+:user:`David Baddeley <David-Baddeley>`
+:user:`Davis Bennett <d-v-b>`
+:user:`Dimitri Papadopoulos Orfanos <DimitriPapadopoulos>`
+:user:`Eduardo Gonzalez <eddienko>`
+:user:`Elliott Sales de Andrade <QuLogic>`
+:user:`Eric Prestat <ericpre>`
+:user:`Eric Younkin <ericgyounkin>`
+:user:`Francesc Alted <FrancescAlted>`
+:user:`Greggory Lee <grlee77>`
+:user:`Gregory R. Lee <grlee77>`
+:user:`Ian Hunt-Isaak <ianhi>`
+:user:`James Bourbeau <jrbourbeau>`
+:user:`Jan Funke <funkey>`
+:user:`Jerome Kelleher <jeromekelleher>`
+:user:`Joe Hamman <jhamman>`
+:user:`Joe Jevnik <llllllllll>`
+:user:`John Kirkham <jakirkham>`
+:user:`Josh Moore <joshmoore>`
+:user:`Juan Nunez-Iglesias <jni>`
+:user:`Justin Swaney <jmswaney>`
+:user:`Mads R. B. Kristensen <madsbk>`
+:user:`Mamy Ratsimbazafy <mratsim>`
+:user:`Martin Durant <martindurant>`
+:user:`Matthew Rocklin <mrocklin>`
+:user:`Matthias Bussonnier <Carreau>`
+:user:`Mattia Almansi <malmans2>`
+:user:`Noah D Brenowitz <nbren12>`
+:user:`Oren Watson <orenwatson>`
+:user:`Pavithra Eswaramoorthy <pavithraes>`
+:user:`Poruri Sai Rahul <rahulporuri>`
+:user:`Prakhar Goel <newt0311>`
+:user:`Raphael Dussin <raphaeldussin>`
+:user:`Ray Bell <raybellwaves>`
+:user:`Richard Scott <RichardScottOZ>`
+:user:`Richard Shaw <jrs65>`
+:user:`Ryan Abernathey <rabernat>`
+:user:`Ryan Williams <ryan-williams>`
+:user:`Saransh Chopra <Saransh-cpp>`
+:user:`Sebastian Grill <yetyetanotherusername>`
+:user:`Shikhar Goenka <shikharsg>`
+:user:`Shivank Chaudhary <Alt-Shivam>`
+:user:`Stephan Hoyer <shoyer>`
+:user:`Stephan Saalfeld <axtimwalde>`
+:user:`Tarik Onalan <onalant>`
+:user:`Tim Crone <tjcrone>`
+:user:`Tobias Kölling <d70-t>`
+:user:`Tom Augspurger <TomAugspurger>`
+:user:`Tom White <tomwhite>`
+:user:`Tommy Tran <potter420>`
+:user:`Trevor Manz <manzt>`
+:user:`Vincent Schut <vincentschut>`
+:user:`Vyas Ramasubramani <vyasr>`
+:user:`Zain Patel <mzjp2>`
+:user:`gsakkis`
+:user:`hailiangzhang <hailiangzhang>`
+:user:`pmav99 <pmav99>`
+:user:`sbalmer <sbalmer>`
```

Comment:
Hi @DON-BRAN. Currently, the `Acknowledgements` are shown like this. 👇🏻
I think this is not reader-friendly. Do you think we can have a bulleted list of items?

<img width="695" alt="Screenshot 2022-12-13 at 00 24 57" src="https://user-images.githubusercontent.com/20305658/207130376-4170e37a-559f-47ee-936e-451f27608bba.png">